src/Libraries/RevitNodesUI/Properties/Resources.resx

```diff
@@ -372,13 +372,14 @@
     <value>The type of the fill pattern.</value>
```

Comment:
This is actually interesting, some of the resource changes are already cherry-picked.. I think I'll have to build to find out but everything else looks good